src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -64,7 +64,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     String password = request.getParameter("password");
     String hashedPassword = BCrypt.hashpw(password, BCrypt.gensalt());
 
-    User user = new User(UUID.randomUUID(), username, hashedPassword, Instant.now());
+    User user = new User(UUID.randomUUID(), username, hashedPassword, Instant.now(), username + " hasn't written a bio yet.");
```

Comment:
The way that you handled bios is really cool

Reply:
haha thanks Padraic! :)